src/main/java/com/twentyonec/ItemsLogger/ItemsLogger.java

```diff
@@ -14,13 +14,15 @@
 import com.twentyonec.ItemsLogger.commands.OpenItemLog;
 import com.twentyonec.ItemsLogger.commands.ViewLogList;
 import com.twentyonec.ItemsLogger.listeners.DeathSave;
+import com.twentyonec.ItemsLogger.listeners.JoinSave;
+import com.twentyonec.ItemsLogger.listeners.QuitSave;
 import com.twentyonec.ItemsLogger.utils.Config;
 import com.twentyonec.ItemsLogger.utils.Storage;
 
 /**
  * Plugin main class
  * 
- * @version 1.0.1 22 October 2020
+ * @version 1.0.1 20 Jan 2021
```

Comment:
You added features. Semantic versioning is `<major>.<minor>.<patch>`
where `major` is breaking changes, `minor` is small new features, and `patch` is bugfixes
```suggestion
 * @version 1.1.0 20 Jan 2021
```